src/custom_keyword.rs

```diff
@@ -161,6 +162,7 @@ macro_rules! impl_parse_for_custom_keyword {
 
 // Not public API.
 #[cfg(not(feature = "parsing"))]
+#[cfg_attr(syn_enable_doc_cfg, doc(cfg(not(feature = "parsing"))))]
```

Comment:
We don't need these attributes on non-public or non-documented APIs, like this one.